client/src/components/EpisodeCard.js

```diff
@@ -0,0 +1,56 @@
+import PropTypes from 'prop-types'
+import styled from 'styled-components/macro'
+
+EpisodeCard.propTypes = {
+  episode: PropTypes.object.isRequired,
+}
+
+export default function EpisodeCard({ episode }) {
+  const { name, episode_number: episodeNumber, overview } = episode
+  return (
+    <Wrapper>
+      <Heading>
+        <span>{episodeNumber}</span>
+        <h4>{name}</h4>
+      </Heading>
+      <p>{overview}</p>
+    </Wrapper>
+  )
+}
+
+const Wrapper = styled.li`
+  display: flex;
+  flex-direction: column;
+  padding: 8px;
+  border-radius: 4px;
+  background: #3c4b5b;
+  margin-bottom: 16px;
+  position: relative;
+
+  p {
+    margin: 0;
+  }
+`
+
```

Comment:
I'm not sure what the teaching style you have had was, but could this not go in a CSS file?